guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -4,12 +4,58 @@
 [id="katello-headline-features"]
 == Headline Features
 
+=== Image-mode host support
+
+Katello has added a new type of host: image-mode hosts.
+Image-mode hosts are standard hosts configured to run a bootc bootable container image at startup.
+Katello {KatelloVersion} allows users to assign bootable container images to an image-mode host to easily deploy identical system configurations, from basic to complex.
```

Comment:
Will this {KatelloVersion} show future Katello versions? Maybe it's better to hard-code, since this is release notes for a specific release?